### PR TITLE
media-video/openshot: bump to 2.3.4 and bump libs

### PR DESCRIPTION
**Pull request by sbraz**
Related bugs:
https://bugs.gentoo.org/show_bug.cgi?id=618410
https://bugs.gentoo.org/show_bug.cgi?id=618408
https://bugs.gentoo.org/show_bug.cgi?id=618404
libopenshot and libopenshot-audio ebuilds did not change except for the addition of Python 3.6.
```diff
--- openshot-2.2.0.ebuild       2017-03-07 12:15:39.924888431 +0100
+++ openshot-2.3.4.ebuild       2017-06-10 16:04:04.678485904 +0200
@@ -4,10 +4,10 @@
 EAPI=6
 
 PYTHON_REQ_USE=xml
-PYTHON_COMPAT=( python3_{4,5} )
+PYTHON_COMPAT=( python3_{4,5,6} )
 DISTUTILS_SINGLE_IMPL=1
 
-inherit distutils-r1 gnome2-utils versionator xdg
+inherit distutils-r1 gnome2-utils versionator xdg-utils
 
 MY_PN="${PN}-qt"
 MY_P="${MY_PN}-${PV}"
@@ -22,7 +22,7 @@
 
 RDEPEND="
        dev-python/PyQt5[svg,webkit,${PYTHON_USEDEP}]
-       media-libs/libopenshot[python,${PYTHON_USEDEP}]
+       >=media-libs/libopenshot-0.1.7[python,${PYTHON_USEDEP}]
        dev-python/httplib2[${PYTHON_USEDEP}]
        dev-python/pyzmq[${PYTHON_USEDEP}]
 "
@@ -33,23 +33,19 @@
 S="${WORKDIR}"
 
 src_prepare() {
+       default
        # prevent setup.py from trying to update MIME databases
        sed -i 's/^ROOT =.*/ROOT = False/' setup.py || die
-       xdg_src_prepare
-       distutils-r1_src_prepare
-}
-
-pkg_preinst() {
-       xdg_pkg_preinst
-       gnome2_icon_savelist
 }
 
 pkg_postinst() {
-       xdg_pkg_postinst
+       xdg_desktop_database_update
+       xdg_mimeinfo_database_update
        gnome2_icon_cache_update
 }
 
 pkg_postrm() {
-       xdg_pkg_postrm
+       xdg_desktop_database_update
+       xdg_mimeinfo_database_update
        gnome2_icon_cache_update
 }

```

### CODE REVIEW

File: media-libs/libopenshot/libopenshot-0.1.7.ebuild [hunk 1]
```diff
@@ -71,9 +74,8 @@ src_configure() {
 }
 
 src_test() {
-	pushd "${BUILD_DIR}/tests" > /dev/null || die
-	./openshot-test || die "Tests failed"
-	popd > /dev/null || die
+	cd "${BUILD_DIR}" || die
+	emake test
```
Comment (mgorny) [inline media-libs/libopenshot/libopenshot-0.1.7.ebuild]: `emake` breaks Ninja, use `cmake-utils_src_make`. Then, is it any different from `default`?

Comment (sbraz) [inline media-libs/libopenshot/libopenshot-0.1.7.ebuild]: Yeah I think it is, because I have to chdir.

Comment (sbraz) [inline media-libs/libopenshot/libopenshot-0.1.7.ebuild]: But I guess I could call cd and then default? I'll try that.

Comment (mgorny) [inline media-libs/libopenshot/libopenshot-0.1.7.ebuild]: I meant `cmake-utils_src_test`.

Comment (sbraz) [inline media-libs/libopenshot/libopenshot-0.1.7.ebuild]: Just so I'm sure: you mean change directory and then call `cmake-utils_src_test`, right?

Comment (mgorny) [inline media-libs/libopenshot/libopenshot-0.1.7.ebuild]: It changes the dir for you.

**Comment by sbraz** [comment]
Putting on hold because of https://bugs.gentoo.org/show_bug.cgi?id=626014

**Comment by sbraz** [comment]
@mgorny I think I've addressed all the issues you raised.

**Comment by sbraz** [comment]
@mgorny I tried yesterday after following the ImageMagick 6 → 7 guide but it's sometimes more than just replacing a keyword with another so I won't risk it. I'll go with the version selector for now.

**Pull request closed without merging**